api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -641,7 +646,7 @@ public void test_maybeSendAccessExpirationEmail_bypassed_is_up_to_date() {
   }
 
   @Test
-  public void test_maybeSendAccessExpirationEmail_expiring_1() {
+  public void test_maybeSendAccessExpirationEmail_expiring_1() throws MessagingException {
```

Comment:
Updated these because `alertUserRegisteredTierWarningThreshold()` changed its signature and now also can throw